spark-jobs/src/test/scala/za/co/absa/enceladus/common/plugin/ControlMetricsPluginSuite.scala

```diff
@@ -90,7 +90,7 @@ class ControlMetricsPluginSuite extends AnyFunSuite {
       Map[String, String]("dummy.1" -> "za.co.absa.NoSuchClass")
         .asJava)
 
-    intercept[IllegalArgumentException] {
+    intercept[Exception] {
```

Comment:
Please revers to previous/higher class or explain why chosen such a generic class?
Same in `[PostProcessorPluginSuite.scala`.

Reply:
Reverted back to IllegalArgumentException, catching exceptions thrown by the function from commons in PluginLoader and throwing IllegalArgumentException instead.